edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Microsoft.Azure.Devices.Edge.Hub.Service.csproj

```diff
@@ -33,6 +33,8 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.3" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.3" />
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="3.1.3" />
+    <PackageReference Include="prometheus-net" Version="3.4.0" />
+	  <PackageReference Include="prometheus-net.AspNetCore" Version="3.4.0" />
```

Comment:
nit: spacing

Reply:
Why do we need a direct ref to prometheus here?

Reply:
UseMetricServer(String); is an extension function added by prometheus-net.AspNetCore, which depends on prometheus-net

Reply:
```suggestion
    <PackageReference Include="prometheus-net.AspNetCore" Version="3.4.0" />
```